diff --git a/buildkernel b/buildkernel
index 0e1b9f4..5a96f8a 100755
--- a/buildkernel
+++ b/buildkernel
@@ -132,7 +132,7 @@ VERBOSITYFLAG=""
 ASKFLAG=""
 ALERTFLAG=""
 PORTAGEINFO=""
-MAKE=""
+MAKE="make LD=ld.bfd"
 # following array variables set by find_all_gpg_keyfile_partitions function
 declare -a GPGUUIDS GPGPARTNAMES GPGDEVNAMES GPGPARTNUMS
 # following array variables set by find_all_luks_partitions function
@@ -1678,11 +1678,11 @@ update_config_if_necessary() {
         if [ "${CONFIGUPDATETYPE}" != "${SILENTUPDATETYPE}" ]; then
             show "Please answer all questions as and if prompted..."
         fi
-        (cd "${LINUXBUILDDIR}" && make "${CONFIGUPDATETYPE}")
+        (cd "${LINUXBUILDDIR}" && ${MAKE} "${CONFIGUPDATETYPE}")
     else
         # non-interactive mode, so we have to go silent
         show "Updating old config using make ${SILENTUPDATETYPE}"
-        (cd "${LINUXBUILDDIR}" && make "${SILENTUPDATETYPE}")
+        (cd "${LINUXBUILDDIR}" && ${MAKE} "${SILENTUPDATETYPE}")
     fi
 }
 conform_config_file() {
@@ -1778,20 +1778,20 @@ allow_user_to_modify_config_graphically() {
     if ((ARG_MENUCONFIG==1)); then
         # force the menuconfig, no matter whether interactive or not
         alertshow "Executing make ${CONFIGTYPE}; select the kernel options you require..."
-        make ${CONFIGTYPE}
+        ${MAKE} ${CONFIGTYPE}
     elif ((ARG_ASK==1)); then
         # interactive mode
         if test_yn "Would you like to modify the configuration using the editor"; then
             show "Executing make ${CONFIGTYPE}; select the kernel options you require..."
-            make ${CONFIGTYPE}
+            ${MAKE} ${CONFIGTYPE}
         else
             show "OK, autoconforming configuration using make ${CONFIGUPDATETYPE} instead..."
-            make ${CONFIGUPDATETYPE}        
+            ${MAKE} ${CONFIGUPDATETYPE}        
         fi
     else
         # non-interactive mode
         show "Autoconforming configuration using make ${SILENTUPDATETYPE}..."
-        make ${SILENTUPDATETYPE}
+        ${MAKE} ${SILENTUPDATETYPE}
     fi
 }
 set_kernel_config() {
@@ -2265,7 +2265,7 @@ process_command_line_options() {
         esac
     done
     # setup make with specified niceness
-    MAKE="nice -n ${ADJUSTMENT} make"
+    MAKE="nice -n ${ADJUSTMENT} ${MAKE}"
     # set verbosity
     if ((ARG_VERBOSE==1)); then
         VERBOSITY+=1
