diff --git a/sx b/sx
index 528750b..bf31b38 100755
--- a/sx
+++ b/sx
@@ -39,5 +39,5 @@ xauth add :"$tty" MIT-MAGIC-COOKIE-1 "$(od -An -N16 -tx /dev/urandom | tr -d ' '
 # Taking advantage of this feature allows us to launch our client directly
 # from a SIGUSR1 handler and avoid the need to poll for server readiness.
 trap 'DISPLAY=:$tty "${@:-$cfgdir/sxrc}"' USR1
-(trap '' USR1 && exec Xorg :"$tty" -keeptty vt"$tty" -noreset -auth "$XAUTHORITY") & pid=$!
+(trap '' USR1 && exec Xorg :"$tty" -keeptty vt"$tty" -noreset -auth "$XAUTHORITY" -ardelay 300 -arinterval 20) & pid=$!
 wait "$pid"
